src/components/ImagesCarouselItem.js

```diff
@@ -13,10 +14,14 @@ import { Image } from './Image';
 export const ImagesCarouselItem = memo(
   ({ navigation, source, message, containerStyle, aspectRatio, refreshInterval }) => {
     const { routeName, params } = source;
+    //for testing
+    console.warn('capitation:', source.captionText);
```

Comment:
remove this before merging

Reply:
solved f90ab7f07ff605af93121f7569894af606fd71e3